tests/cypress/integration/actions_tasks3/case_109_dummy_cloud_storage.js

```diff
@@ -0,0 +1,196 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+const {
+    cloudStoragesDummyDataGoogleStorage,
+    cloudStoragesDummyDataAzureContainer,
+    cloudStoragesDummyDataAWSBucket,
+} = require('../../support/dummy-data');
+
+context('Dummy Cloud storages.', { browser: '!firefox' }, () => {
+    const caseId = '109';
+    const imageFolder = '../integration/actions_tasks3/assets/case_109';
+
+    function testCreateDummyStorage(dummyCS) {
+        cy.intercept('GET', 'api/v1/cloudstorages?page_size=12&page=1', dummyCS).as('createCS');
+        cy.contains('.cvat-header-button', 'Cloud Storages').should('be.visible').click();
+        cy.wait('@createCS').its('response.statusCode').should('eq', 200);
+        cy.get('.cvat-cloud-storage-item-empty-preview').should('have.length', 1);
+    }
+
+    function testCheckAndCloseNotification() {
+        // Check and close notifications
+        cy.get('.cvat-notification-notice-fetch-cloud-storage-status-failed').should('have.length', 1);
+        cy.closeNotification('.cvat-notification-notice-fetch-cloud-storage-status-failed');
+        cy.get('.cvat-notification-notice-fetch-cloud-storage-preview-failed').should('have.length', 1);
+        cy.closeNotification('.cvat-notification-notice-fetch-cloud-storage-preview-failed');
+    }
+
+    function testCSValues({
+        status, id, description, provider,
+    }) {
+        cy.get('.cvat-cloud-storage-item').should('have.length', 1);
+        cy.get('.cvat-cloud-storage-item')
+            .should('contain', `Status: ${status}`);
+        if (description) {
+            cy.get('.cvat-cloud-storage-item')
+                .should('contain', `#${id}: ${description}`);
```

Comment:
Is a `displayName` instead of a `description` implied?

Reply:
Sure. Done.